_posts/blog/2017-02-23-security-performance-implications.md

```diff
@@ -0,0 +1,158 @@
+
+The purpose of this two part series was to measure the performance impact of
+various security configurations on a cluster running Apache Accumulo’s
+continuous ingest suite. The tests were performed using Amazon Web
+Services (AWS), Hortonworks Data Platform 2.4 and Accumulo 1.7. Each of
+the five different security settings in Accumulo 1.7 was tested
```

Comment:
HDFS was configured to use kerberos for authentication and had service level authorization on.  Other than that, no other settings (https, rpc protection, data transfer encryption, etc) were enabled.  

I can add that clarification to the post.

Enabling additional security settings will be called out in the "Future Work" section of Part 2.